test_scripts/API/PerformAudioPassThru/025_ATF_P_PerformAudioPassThru_UI_SUCCESS_TTS_success_false_true.lua

```diff
@@ -0,0 +1,267 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PerformAudioPassThru] Speak (<errorCode>) + UI.PerformAudioPassThru (SUCCESS)
```

Comment:
@maria-mih, I didn't manage to  find requirement that specify success result code of Speak, but script is verifying success true also.